aspnetcore-features/CustomAttributes/TestProject/PrintAttributesUnitTest.cs

```diff
@@ -0,0 +1,43 @@
+using CustomAttributes;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace TestProject
+{
+    [TestClass]
+    public class PrintAttributesUnitTest
+    {
+        [TestMethod]
+        public void WhenCorrectInput_ThenCorrectSingleAttOutput()
+        {
+            bool result = true;
+
+            try
+            {
+                CustomAttributeHelper.GetTaskAttribute(typeof(TaskGroup));
+            }
+            catch
+            {
+                result = false;
+            }
```

Comment:
Does this ever get caught? Why guard it with a catch block? I don't see how it can ever get to it.